robottelo/api/utils.py

```diff
@@ -2,10 +2,19 @@
 """Module containing convenience functions for working with the API."""
 import time
 
+from fauxfactory import gen_string
 from inflector import Inflector
-from nailgun import entities
+from nailgun import entities, entity_mixins
 from robottelo import ssh
 from robottelo.constants import PERMISSIONS_WITH_BZ
+from robottelo.config import settings
+from robottelo.constants import (
```

Comment:
please, put in one statement with `from robottelo.constants import PERMISSIONS_WITH_BZ`